binja/krf.py

```diff
@@ -0,0 +1,130 @@
+import logging
+
+log = logging.getLogger(__name__)
+
+import binaryninja as bn
+from binaryninja.mediumlevelil import MediumLevelILOperation
+
+
+class KRFAnalysis(object):
```

Comment:
nit: Drop `object` from the class specifier; I'm pretty sure Python 3 doesn't need it:

```python
class KRFAnalysis:
  ...
```